tests/e2e/standalone/init_test.go

```diff
@@ -62,6 +62,15 @@ func TestStandaloneInit(t *testing.T) {
 		require.Contains(t, output, "both --image-registry and --from-dir flags cannot be given at the same time")
 	})
 
+	t.Run("init should error out if container runtime is not valid", func(t *testing.T) {
+		// Ensure a clean environment
+		must(t, cmdUninstall, "failed to uninstall Dapr")
+
+		output, err := cmdInit(daprRuntimeVersion, "--container-runtime", "invalid")
+		require.Error(t, err, "expected error if container runtime is invalid")
+		require.Contains(t, output, "Invalid container runtime")
+	})
+
 	t.Run("init", func(t *testing.T) {
```

Comment:
@shubham1172 same for line 121 .... do not skip .... just return ... since without that verify binaries and verify configs will not be run for slim init ...

Reply:
Good catch!